src/sentry/web/frontend/twofactor.py

```diff
@@ -142,12 +146,22 @@ def handle(self, request):
                 return self.perform_signin(request, user, used_interface)
             self.fail_signin(request, user, form)
 
-        # If a challenge and response exists, validate
+        # check if webauthn-login feature flag is enabled for frontend
+        is_webauthn_signin_ff_enabled = False
+        orgs = self._get_org_from_user(user)
+        if any(
```

Comment:
I'm wondering if we should make this a user-based feature flag instead of an org-based one. I'm unsure if users that belong to many orgs would be impacted negatively by the performance on this.

Reply:
via slack thread in discuss-backend, this should be fine.